Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -118,28 +154,51 @@ void doChargeDepositionShapeN (const GetParticlePosition& GetPosition,
 #endif
       {
 #if defined(AMREX_USE_GPU)
-        long ip = threadIdx.x + (long)blockIdx.x*blockDim.x;
+        const int bin_id = blockIdx.x;
+        const unsigned int bin_start = offsets_ptr[bin_id];
+        const unsigned int bin_stop = offsets_ptr[bin_id+1];
+
+        if (bin_start == bin_stop) { return; }
+
+        amrex::Box buffer_box;
+        {
+          ParticleReal xp, yp, zp;
+          GetPosition(bin_start, xp, yp, zp);
+#if (AMREX_SPACEDIM == 3)
+          IntVect iv = IntVect(int(amrex::Math::floor((xp-plo[0])*dxiarr[0])),
+                               int(amrex::Math::floor((yp-plo[1])*dxiarr[1])),
+                               int(amrex::Math::floor((zp-plo[2])*dxiarr[2])));
+#elif (AMREX_SPACEDIM == 2)
+          IntVect iv = IntVect(
+                               int(amrex::Math::floor((xp-plo[0])*dxiarr[0])),
+                               int(amrex::Math::floor((zp-plo[1])*dxiarr[1])));
```

Comment:
The 1D definition for `iv` was missing.
```suggestion
                               int(amrex::Math::floor((zp-plo[1])*dxiarr[1])));
#elif (AMREX_SPACEDIM == 1)
          IntVect iv = IntVect(int(amrex::math::floor((zp-plo[0])*dxiarr[0])));
                              
```